autoscale/src/main/java/com/sequenceiq/periscope/controller/AlertController.java

```diff
@@ -229,14 +224,34 @@ private LoadAlertResponse createLoadAlertResponse(LoadAlert loadAlert) {
         return loadAlertResponseConverter.convert(loadAlert);
     }
 
-    private void validateTimeAlert(Long clusterId, Optional<Long> alertId, TimeAlertRequest json) throws ParseException {
-        validateAlertForUpdate(clusterId, alertId, AlertType.TIME);
-        dateService.validateTimeZone(json.getTimeZone());
-        dateService.getCronExpression(json.getCron());
+    private void validateTimeAlert(Long clusterId, Optional<Long> alertId, TimeAlertRequest json) {
+        alertId.ifPresent(alert -> validateAlertForUpdate(clusterId, alert, AlertType.TIME));
+        try {
+            dateService.validateTimeZone(json.getTimeZone());
+            dateService.getCronExpression(json.getCron());
+        } catch (ParseException parseException) {
+            throw new BadRequestException(parseException.getMessage(), parseException);
+        }
     }
 
-    private void validateLoadAlert(Long clusterId, Optional<Long> alertId, LoadAlertRequest json) throws ParseException {
-        validateAlertForUpdate(clusterId, alertId, AlertType.LOAD);
+    private void validateLoadAlert(Long clusterId, Optional<Long> alertId, LoadAlertRequest json) {
+        Cluster cluster = clusterService.findById(clusterId);
+        alertId.ifPresentOrElse(
+                updateAlert -> {
+                    validateAlertForUpdate(clusterId, updateAlert, AlertType.LOAD);
+                }, () -> {
+                    if (!cluster.getTunnel().useClusterProxy()) {
+                        throw new BadRequestException(String.format("Cluster '%s' is not configured with Cluster Proxy Tunnel, " +
+                                "Cluster Tunnel is %s. Load Based Scaling not supported", cluster.getStackCrn(), cluster.getTunnel()));
+                    }
+
+                    String hostGroup = json.getScalingPolicy().getHostGroup();
```

Comment:
Do we need to check for the existence of the hostGroup in the cluster?

Reply:
we would also need the hostGroup services validation, which can be added based on our other discussion.